### ISSUE TITLE
Invalid expiration date

### ISSUE THREAD

**Issue by Mneroen** [braintree/braintree-web#238]
I have found a bug with the dropdown expiration date in hosted fields.

The default value of the expiration month is always 1. If the card expiration date is 01/2017, 01/2018 or 01/2019... the user will only change the year and the expiration month will be invalid. The expiration month will only be valid if the user change the month to 02 then back to 01.

If the current date will be 01/2017 and the card expiration date is 01/2017 the user have to change the month to 01->02->01 and the year to 2017->2018->2017

**Comment by crookedneighbor** [comment]
Thanks for reporting this. It looks like if you create the drop down inputs _without_ a placeholder field, this happens. We should fix this behavior, but in the meantime, you can create your fields like this to solve the problem:

```js
expirationMonth: {
  selector: '#your-expiration-month-div',
  placeholder: '-- Expiration month --',
  select: {
    options: [
      '1 - Jan',
      '2 - Feb',
      '3 - Mar',
      '4 - Apr',
      '5 - May',
      '6 - Jun',
      '7 - Jul',
      '8 - Aug',
      '9 - Sep',
      '10 - Oct',
      '11 - Nov',
      '12 - Dec'
    ]
  }
},
expirationYear: {
  selector: '#your-expiration-year-div',
  placeholder: '-- Expiration year --',
  select: true
},
```

CC @EvanHahn

**Comment by skunkworks** [comment]
@Mneroen Thanks again for reporting this. We've patched this bug and will be releasing the fix with the next version. We'll leave the issue open until then.

**Comment by skunkworks** [comment]
@Mneroen This issue has been fixed in [3.7.0](https://github.com/braintree/braintree-web/releases/tag/3.7.0).
